lib/matplotlib/ticker.py

```diff
@@ -829,13 +829,13 @@ class LogFormatter(Formatter):
     """
     Format values for log axis.
     """
-    def __init__(self, base=10.0, labelOnlyBase=True):
+    def __init__(self, base=10.0, label_only_base=True):
         """
         `base` is used to locate the decade tick, which will be the only
-        one to be labeled if `labelOnlyBase` is ``True``.
+        one to be labeled if `label_only_base` is ``True``.
         """
         self._base = base + 0.0
-        self.labelOnlyBase = labelOnlyBase
+        self.label_only_base = label_only_base
```

Comment:
This is also a public API break which should be patched over with a property, but should also wait for changing the parameter name in the `__init__`